src/pe_reports/metrics.py

```diff
@@ -12,25 +22,571 @@ def __init__(self, start_date, end_date, org_uid):
+        ]
+        c6_df_2 = c6_df_2.rename(columns={"create_time": "modified_date"})
+        view_df_2 = hibp_df[
+            [
+                "breach_name",
+                "modified_date",
+                "description",
+                "breach_date",
+                "password_included",
+                "email",
+            ]
+        ]
+        view_df_2 = view_df_2.append(c6_df_2, ignore_index=True)
+
+        breach_df = view_df_2.groupby(
+            [
+                "breach_name",
+                "modified_date",
+                "description",
+                "breach_date",
+                "password_included",
+            ],
+            as_index=False,
+        ).agg({"email": ["count"]})
+
+        breach_df.columns = breach_df.columns.droplevel(1)
+        breach_df = breach_df.rename(columns={"email": "number_of_creds"})
+        breach_df = breach_df[
+            [
+                "breach_name",
+                "breach_date",
+                "modified_date",
+                "password_included",
+                "number_of_creds",
+            ]
+        ]
+        breach_det_df = breach_df.rename(columns={"modified_date": "update_date"})
+
+        if len(breach_det_df) > 0:
+            breach_det_df["update_date"] = breach_det_df["update_date"].dt.strftime(
+                "%m/%d/%y"
+            )
+            breach_det_df["breach_date"] = pd.to_datetime(
+                breach_det_df["breach_date"]
+            ).dt.strftime("%m/%d/%y")
+
+        breach_det_df = breach_det_df.rename(
+            columns={
+                "breach_name": "Breach Name",
+                "breach_date": "Breach Date",
+                "update_date": "Date Reported",
+                "password_included": "Password Included",
+                "number_of_creds": "Number of Creds",
+            }
+        )
+
+        return breach_det_df
+
+    def password(self):
+        """Return total number of credentials with passwords."""
+        pw_creds_csg = len(
+            self.query_cyberSix_creds[self.query_cyberSix_creds["password_included"]]
+        )
+        pw_creds_hibp = len(
+            self.query_hibp_view[self.query_hibp_view["password_included"]]
+        )
+        return pw_creds_csg + pw_creds_hibp
+
     def total(self):
-        """Return total number of credentials."""
+        """Return total number of credentials found in breaches."""
         df_cred_csg = self.query_cyberSix_creds.shape[0]
         df_cred_hibp = self.query_hibp_view.shape[0]
         return df_cred_csg + df_cred_hibp
 
-    # TODO the following functions correspond to functions at report_generator.py
-    # TODO and will be added in follow up PR found here: https://github.com/cisagov/pe-reports/pull/99
-    def password(self):
-        """Return total number of credentials with passwords."""
 
-    def breached(self):
-        """Return total number of breached credentials."""
+class Domains_Masqs:
+    """Domains Masquerading Class."""
```

Comment:
Unnecessary capitalization:
```suggestion
    """Domains Masquerading class."""
```